### ISSUE TITLE
Problem inserting emoji as a file/folder name

### ISSUE THREAD

**Issue by amoshydra** [nus-mtp/lecture-note-2.0#314]
Test case: Rename a file with 🔥💕💯🌹

When renaming a file to use emoji, the browser will automatically wrap the emoji with additional span tag
```html
<p data-v-0073b85a="" contenteditable="true" tabindex="2" class="inline-block">
    <span style="color: rgb(84, 84, 84); font-family: arial, sans-serif; font-size: small; white-space: normal; background-color: rgb(255, 255, 255);">🔥💕💯🌹</span>
</p>
```

The HTML of the `span` will then be used as the parameter for renaming a file.

e.g. 
```javacsript
File.rename('<span style="color: rgb(84, 84, 84); font-family: arial, sans-serif; font-size: small; white-space: normal; background-color: rgb(255, 255, 255);">🔥💕💯🌹</span>')
```

Currently, the filename failed to be inserted due to #304. But this bug is related to `FileManager.vue`

**Comment by jiayingy** [comment]
The issue doesn't lie with emojis. The problem is with it doesn't paste with plain text

**Comment by jiayingy** [comment]
Fixed at e0f84301
